### ISSUE TITLE
the regex input field can't be navigated with screenreader

### ISSUE THREAD

**Issue by vojtapolasek** [firasdib/Regex101#1505]
## Bug Description
I am a screenreader user, using Orca screenreader. When I type characters in the regex input field, I can't read them later when using arrows. They are also not announced when deleted.

### Reproduction steps
1. Turn on Orca screenreader, click into the "regex" edit box and start writing characters.
2. use left arrow to go back through characters.
3. alternatively, use backspace to delete some characters.

### Expected Outcome
Characters are read out loud.

### Browser
Firefox 85.0

### OS
Fedora 32

**Comment by firasdib** [comment]
Why can't you read them? I have not changed anything with regards to that, so I'm not sure what would cause that issue.

**Comment by vojtapolasek** [comment]
I have no idea. Aren't you doing some automatic formattig or something during entering of characters into the edit box? Aren't you somehow changing input on the fly? Just guessing.

**Comment by firasdib** [comment]
No, the text never changes. If it worked before it should work now. Has the software recently been updated? Can you describe the problem in greater detail?


Vänliga hälsningar / Best regards,

Firas Dib

**Comment by vojtapolasek** [comment]
Hi, I tried with several versions of Orca screenreader for Linux and the problem is the same.
In normal edit fields, when I type some text, I can then use arrows to move cursor backwards. As the cursor crosses characters, they are read. Also when I deleta a character with backspace, it is announced. In this field, it does not work at all. It holds for the field for "regex" as well as for "test string". But for example in the field for "searching the reference", it works as expected.

**Comment by firasdib** [comment]
@vojtapolasek Thanks for testing it out. Do you mind trying it out on https://codemirror.net/? Does that editor also have the same behavior?

**Comment by vojtapolasek** [comment]
Yes, the problem is the same.

**Comment by firasdib** [comment]
Thanks for verifying, then its an issue I cannot solve in the underlying library. Is it also present in the new version of the library, https://codemirror.net/6/?

**Comment by vojtapolasek** [comment]
No, in this text field it works as expected.

**Comment by firasdib** [comment]
Great. When that gets ready for public use, the issue will be resolved on regex101 :) 


Vänliga hälsningar / Best regards,

Firas Dib

**Comment by firasdib** [comment]
@vojtapolasek is this still a problem?

**Comment by firasdib** [comment]
This should be resolved with the upcoming update using Codemirror 6.
